src/pages/LandingPages/CorporateCampaign/CCLandingPage.vue

```diff
@@ -560,8 +560,9 @@ export default {
 	},
 	computed: {
 		pageTitle() {
-			// TODO: add field on Contentful Page for this
-			return this.pageData?.page?.pageLayout?.name;
+			const layoutTitle = this.pageData?.page?.pageLayout?.pageTitle;
+			const pageTitle = this.pageData?.page?.pageTitle ?? 'Loans that change lives';
+			return layoutTitle || pageTitle;
```

Comment:
Would we prefer pageTitle first and fallback to layoutTitle?

Reply:
Eventually there will be multiple layouts and an experiment or other config could control which layout is shown, thus the Layout takes precedent.